Documentation/network/lb-ipam.rst

```diff
@@ -395,6 +395,27 @@ loadBalancerClass               Feature
 If the ``.spec.loadBalancerClass`` is set to a class which isn't handled by Cilium's LB IPAM, 
 then Cilium's LB IPAM will ignore the service entirely, not even setting a condition in the status. 
 
+By default, if the ``.spec.loadBalancerClass`` field is not set, Cilium's LB IPAM will assume it can 
+allocate IPs for the service. If this isn't the desired behavior, you can configure LB-IPAM to only 
```

Comment:
Suggestion: can we expand this to specify where it will allocate IPs from for a service if we don't have a load balancer class set (and `requireLBClass=false`)